frontend/src/components/CoronaProvinceComponents/CoronaProvinceSelect.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components/macro'
+
+export default function CoronaProvinceSelect({provinces, setSelectedProvince}) {
+
+    const selectProvince = (e) => setSelectedProvince(e.target.value)
+
+    return (
+
+        <SelectWrapper>
+            <p>Wähle Deine Region aus:</p>
+            <ProvinceSelector required defaultValue={''} onChange={e => selectProvince(e)}>
```

Comment:
```suggestion
    return (

        <SelectWrapper>
            <p>Wähle Deine Region aus:</p>
            <ProvinceSelector required defaultValue={''} onChange={e => setSelectedProvince(e.target.value)}>
```

I would use setSelectedProvince directly. Could be confusing what is the difference between selectProvince and setSelectedProvince.

Reply:
changes I have made 🦑